### PR TITLE
expand guidelines for blog post submissions

### PR DESCRIPTION
**Pull request by fmichonneau**
draft of expanded recommendations for file naming when contributing to our website

### CODE REVIEW

File: topic_folders/communications/guides/submit_blog_post.md [hunk 1]
```diff
@@ -28,6 +29,32 @@ Read below and find out how to contribute a blog post to
...
+  are not.
+* Try to avoid using shorthand, acronyms, or contractions in the file names (and
+  by extension in the permalinks).
+* Favor using the actual numbers in the file names and permakinks instead: use
+  `12` rather than `twelve`.
+* When relevant, include `carpentries` in the file names and permalinks: favor
+  `new-carpentries-team-member` over `new-team-member`.
+* Use 3 to 6 words to compose the file names (and permalinks) and the headings
+  on your page and blog posts.
+* For titles, we recommend ...
```
Comment (fmichonneau) [inline topic_folders/communications/guides/submit_blog_post.md:45]: @kmomar please complete

File: topic_folders/communications/guides/submit_blog_post.md [hunk 2]
```diff
@@ -42,8 +42,8 @@ Read below and find out how to contribute a blog post to
   `new-carpentries-team-member` over `new-team-member`.
 * Use 3 to 6 words to compose the file names (and permalinks) and the headings
   on your page and blog posts.
-* For titles, we recommend ...
-* For headings, we recommend ...
+* For titles, we recommend making sure the title is descriptive and uses no more than 5-7 words. EX: How We Use Feedback at The Carpentries
```
Comment (fmichonneau) [inline topic_folders/communications/guides/submit_blog_post.md:45]: should we mention using title case here?

File: topic_folders/communications/guides/submit_blog_post.md [hunk 3]
```diff
@@ -42,8 +42,8 @@ Read below and find out how to contribute a blog post to
...
+* For headings, we recommend a full sentence that succinctly describes the contents of the blog. EX: Read on to learn about our strategic plan progress through Q3 2020 (July - September)
```
Comment (fmichonneau) [inline topic_folders/communications/guides/submit_blog_post.md:46]: Could we maybe add a few more recommendations here like "Start with an actionable verb for posts that involve the community, e.g., 'Become a mentor or mentee for the first cycle of 2021 Mentoring Groups!'", or "Avoid starting the heading with 'in this blog post we...'", anything else?

**Comment by fmichonneau** [comment]
@kmomar @serahrono is this ready to be merged?

**Comment by kmomar** [comment]
@fmichonneau in my opinion yes, but I will leave it for @serahrono to confirm.

**Review by serahrono — CHANGES_REQUESTED** [review]
thank you for working on this @fmichonneau and @kmomar. Questions for you, 
- are there short guides / tutorials from our resource bank or elsewhere that we can point to that walk people through how to write good blog post headlines? I like that this is a clear set of dos and don'ts, and would like us to qualify some of the best practice recommendations / or at least help make it easier for people to do the right thing
- because we are recommending title case use for our blog post titles, I would like us to go a step further and state what convention we recommend using [from this list](https://titlecaseconverter.com/rules/). Once we know, we would be in a good position to point people to the [titlecaseconverter tool](https://titlecaseconverter.com) as a good way to make it easier to get it right, and correct any inconsistencies during the review process.

**Comment by kmomar** [comment]
Hi @serahrono: regarding your points above,
- From my cursory search, most of the blog post headline guidelines I could find were for increasing clicks and traffic, which is not necessarily what I think we are looking for. I am happy to search more, but wanted to report back on that.
- I think that we should use AP style, if only for the reason that I am most familiar with it and as one of the team members who works on the blog it would help my work, but if there are salient criticisms then I am happy to change that.

**Comment by maneesha** [comment]
This had been pending review from Serah a few months ago.  @kmomar  @acrall  given the notes above, is this ok to merge?

**Comment by acrall** [comment]
@maneesha, @kmomar and I discussed, and decided to merge

**Pull request merged**